scripts/cot/botania/flowers/functional/amuileria_kaerunea.zs

```diff
@@ -0,0 +1,104 @@
+/*
+Amuileria kaerunea (Aquilegia caerulea + Kaminari[thunder]) - crafting with lightning
+*/
+
+#loader contenttweaker
+
+import crafttweaker.damage.IDamageSource;
+import crafttweaker.data.IData;
+import crafttweaker.item.IItemStack;
+import crafttweaker.util.Math;
+import crafttweaker.world.IBlockPos;
+import crafttweaker.world.IWorld;
+import mods.contenttweaker.Color;
+import mods.contenttweaker.VanillaFactory;
+import mods.randomtweaker.cote.BlockAdded;
+import mods.randomtweaker.cote.ISubTileEntity;
+import mods.randomtweaker.cote.ISubTileEntityFunctional;
+import mods.randomtweaker.cote.SubTileEntityInGame;
+
+static recipesLigthningFlower as IItemStack[string] = {
+'item.appliedenergistics2.material.certus_quartz_crystal': <item:appliedenergistics2:material:1>,
+'item.appliedenergistics2.material.purified_certus_quartz_crystal': <item:appliedenergistics2:material:1>,
+'item.appliedenergistics2.material.certus_quartz_crystal_charged': <item:appliedenergistics2:material:1>,
+} as IItemStack[string];
```

Comment:
Please add this recipes at least into crafting hints by `scripts.jei.crafting_hints.add1to1()`.
I dont know how to do that, but i think you need some kind of a trick to have recipe list available from both places - `#loader contenttweaker` and default loader. Ask me if you will not be able to find solution.

Reply:
Same as ZenUtils LateSetCoTFunction, RandomTweaker custom botania flowers also support it.
`<cotSubTile:name>`